ez_setup.py

```diff
@@ -120,8 +120,7 @@ def archive_context(filename):
 
 def _do_download(version, download_base, to_dir, download_delay):
     """Download Setuptools."""
-    egg = os.path.join(to_dir, 'setuptools-%s-py%d.%d.egg'
-                       % (version, sys.version_info[0], sys.version_info[1]))
+    egg = os.path.join(to_dir, 'setuptools-{}-py{:d}.{:d}.egg'.format(version, sys.version_info[0], sys.version_info[1]))
```

Comment:
PEP8 says that the line should not be too long, so would be nice to break this line up.